src/components/data-management/SamplesTable.jsx

```diff
@@ -2,6 +2,7 @@
 import React, {
   useEffect, useState, forwardRef, useImperativeHandle,
 } from 'react';
+import Storage from '@aws-amplify/storage';
```

Comment:
Why are we importing storage? We replaced it with the signed urls and axios.

Reply:
We still use this to download the public dataset directly from S3:
https://github.com/hms-dbmi-cellenics/ui/blob/d120f71a700aa61c74d6bdc471a946c90126660f/src/components/data-management/SamplesTable.jsx#L268

Reply:
We might want a new endpoint for this, one without authentication, to download the public datasets. Should I create a new ticket?

Reply:
why would you want to download the datasets without authentication?

Reply:
Sorry - I mean without authorization, because these public datasets are not related to any experiment ids.